packages/nimble-components/src/mapping/base/index.ts

```diff
@@ -0,0 +1,22 @@
+import { attr } from '@microsoft/fast-element';
+import { FoundationElement } from '@microsoft/fast-foundation';
+import type { MappingKey } from './types';
+
+/**
+ * Base class for mapping configuration elements
+ */
+export abstract class Mapping extends FoundationElement {
+    /**
+     * The data value that is mapped to another representation
+     */
+    @attr()
+    public key?: MappingKey;
+
+    /**
+     * In the case of a mapping that maps to text, this is that mapped text.
+     * For mappings to non-textual representations, this is a display string
```

Comment:
It's not displayed right? That's the whole distinction?

```suggestion
     * For mappings to non-textual representations, this is a string
```

Reply:
Well, no, it could be. For the icon/spinner mappings, the `text` value appears in the tooltip, and also next to the icon/spinner in the group header. So it should be formatted appropriately for display.

Reply:
"For mappings to non-textual representations, this is a string that can be displayed in contexts such as group headers or tooltips and used as an accessible label"